src/Divante/MagentoIntegrationBundle/Infrastructure/Common/AbstractIntegratedObjectRepository.php

```diff
@@ -75,7 +75,7 @@ public function getObjects(array $objectsId, IntegrationConfiguration $configura
                 ]
             )
         );
-
+        $listing->setObjectTypes([AbstractObject::OBJECT_TYPE_VARIANT, AbstractObject::OBJECT_TYPE_OBJECT]);
```

Comment:
Also needs to me added in function `getAllObjects`

Reply:
added